__tests__/integration/create-token.test.js

```diff
@@ -121,7 +121,7 @@ describe('create token', () => {
 
     expect(response.status).toBe(200);
     expect(response.body.success).toBe(false);
-    expect(response.body.error).toContain('Change address');
```

Comment:
The wallet was returning the lib error message, since the lib changed the validation error the message changed.
Do you think we should track the messages returned to the user on the lib or the headless?

Reply:
I think this deserves to be tracked in both places:
- On the lib, since it serves many of our products and partners;
- On the headless, because this is the final result for the end user

It's a code duplication, but when we are asserting the results it feels more like a redundancy than a code smell. Do you agree?

( by the way, this code is already OK again )